theme_kea/views/snippets/s_image_text.xml

```diff
@@ -3,19 +3,24 @@
     <template id="s_image_text" inherit_id="website.s_image_text">
         <!-- Section -->
         <xpath expr="//section" position="attributes">
-            <attribute name="class" add="o_cc o_cc4 pt144" remove="pt32" separator=" "/>
-            <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/02_001","flip":["y"]}</attribute>
+            <attribute name="class" add="o_cc o_cc2 pt232" remove="pt32" separator=" "/>
+            <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/Wavy_22","flip":["y"]}</attribute>
         </xpath>
         <!-- Shape -->
         <xpath expr="//div[hasclass('container')]" position="before">
-            <div class="o_we_shape o_web_editor_Origins_02_001 o_we_flip_y"/>
+            <div class="o_we_shape o_web_editor_Wavy_22 o_we_flip_y"/>
+        </xpath>
+        <!-- Image -->
+        <xpath expr="//img" position="attributes">
+            <attribute name="src">/web_editor/shape/theme_kea/s_image_text.svg?c1=%23d46926</attribute>
         </xpath>
         <!-- Title & Paragraph -->
         <xpath expr="(//div[hasclass('col-lg-6')])[2]" position="replace">
             <div class="col-lg-6 pt16 pb16">
-                <h2>Enter in a new world of control.</h2>
-                <p>Control every electronic device of your house. The coffee machine, the television, the thermostat, everything.</p>
-            </div>
+                <h2>Enter in a new world</h2>
+                    <p>Write one or two paragraphs describing your product or services. To be successful your content needs to be useful to your readers.</p>
```

Comment:
Don't put back the default text. Just remove the previous customization.
No need to replace the whole column if you just want to replace the title.